### PR TITLE
doc: fix broken TOC and expand examples

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
**What**:
- Generated toc to remove links

**Why**:
- Links are still present in the Readme, which are stale

**How**:
- Ran precommit hook to auto generate toc

**Checklist**:

- [X] Documentation
- [ ] Tests
- [X] Ready to be merged 
- [X] Added myself to contributors table

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,13 +168,11 @@ Thanks goes to these wonderful people
 ([emoji key](https://github.com/kentcdodds/all-contributors#emoji-key)):
 
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
-
 <!-- prettier-ignore -->
```
Comment (M-ZubairAhmed) [inline README.md]: I did not remove this line, this happened with pre commit hook

Comment (machour) [reply README.md]: That must be prettier, thanks for the clarification :)

File: .all-contributorsrc [hunk 2]
```diff
@@ -171,6 +171,17 @@
...
+    {
+      "login": "M-ZubairAhmed",
+      "name": "Md Zubair Ahmed",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/17708702?v=4",
+      "profile": "https://in.linkedin.com/in/mzubairahmed",
+      "contributions": [
+        "code",
+        "bug",
+        "test"
+      ]
```
Comment (machour) [inline .all-contributorsrc]: Use only "doc" and "bug" for now please, I'll merge this PR in right away so that you'll be able to add "code" in  #78 ;)

**Comment by M-ZubairAhmed** [comment]
i havent changed any code in this branch expects added contributor. I dont know travis is failing

File: .all-contributorsrc (hunk 2 above)
Comment (M-ZubairAhmed) [reply .all-contributorsrc]: sure

**Comment by M-ZubairAhmed** [comment]
changes done 👍

**Comment by machour** [comment]
That Travis CI error was weird, provided that you only changed doc files.
Fixed it by disabling the lint check for the concerned function in https://github.com/jfmengels/all-contributors-cli/commit/99871508e873225fc3916785a83fe0865fb3aa21

/cc @kentcdodds if you have an explanation once Santa is gone 🎅

**Pull request merged**